Code/Framework/AzToolsFramework/AzToolsFramework/Thumbnails/LoadingThumbnail.cpp

```diff
@@ -14,39 +14,19 @@ namespace AzToolsFramework
 {
     namespace Thumbnailer
     {
-        static constexpr const int LoadingThumbnailSize = 128;
+        static constexpr const int LoadingThumbnailSize = 1;
 
         //////////////////////////////////////////////////////////////////////////
         // LoadingThumbnail
         //////////////////////////////////////////////////////////////////////////
-        static constexpr const char* LoadingIconPath = "Assets/Editor/Icons/AssetBrowser/in_progress.gif";
-
         LoadingThumbnail::LoadingThumbnail()
             : Thumbnail(MAKE_TKEY(ThumbnailKey))
         {
-            auto absoluteIconPath = AZ::IO::FixedMaxPath(AZ::Utils::GetEnginePath()) / LoadingIconPath;
-            m_loadingMovie.setFileName(absoluteIconPath.c_str());
-            m_loadingMovie.setCacheMode(QMovie::CacheMode::CacheAll);
-            m_loadingMovie.setScaledSize(QSize(LoadingThumbnailSize, LoadingThumbnailSize));
-            m_loadingMovie.start();
-            m_pixmap = m_loadingMovie.currentPixmap();
+            m_pixmap = QPixmap(LoadingThumbnailSize, LoadingThumbnailSize);
+            m_pixmap .fill(Qt::black);
```

Comment:
```suggestion
            m_pixmap .fill(Qt::magenta );
```
Should we make it an obviously unloaded looking color, or is that too garish?

Reply:
Met with reps from UX. They recommended not using color as an indicator for accessibility reasons. Black was preferred, possibly adding 50% white loading text or a simple icon at the center of the image. My only reservation about text is that the image is scaled for large and small thumbnails and the text might not be legible at a smaller size.